app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddBusinessPage.scala.html

```diff
@@ -33,14 +33,20 @@
   hmrcAddToAList : HmrcAddToAList
 )
 
-@(form: Form[_], businessEntity: List[BusinessEntity], businessAdded: Boolean = false, businessRemoved: Boolean = false)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(form: Form[_], businessEntity: List[BusinessEntity], businessAdded: Boolean = false, businessRemoved: Boolean = false, newlyAddedEoriOpt: Option[String] = None, removedAddedEoriOpt: Option[String] = None)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
 @titleKey = @{messages("addBusiness.title")}
 
+@addedBannerContent = {
+    <h3 class="govuk-notification-banner__heading">@Html(messages("addBusiness.businessAdded.success.bannerTitle"))</h3>
```

Comment:
preference thing, but I feel a getOrElse is cleaner then a map here. Happy to go with majority decision!

Reply:
i prefer the way Yonis has done it because the element will not display if the value is not there. Can discuss further if you like 👍

Reply:
ahh thats a very good point, LGTM then!